Sources/Interaction/Manipulators/MouseCameraAxisRotateManipulator/index.js

```diff
@@ -128,7 +129,12 @@ function vtkMouseCameraAxisRotateManipulator(publicAPI, model) {
         const fpDist = vec3.length(newFp);
 
         // Note it normalizes the vector to be rotated
-        const result = vtkMath.rotateVector(newFp, v2, rAngle);
+        const result = [...newFp];
+        vtkMatrixBuilder
+          .buildFromRadian()
+          .identity()
```

Comment:
couldn't you get rid of identity matrix ?

Reply:
Indeed. As the constructor already set the matrix to the identity.